src/OpenTelemetry/Trace/SimpleActivityProcessor.cs

```diff
@@ -91,6 +91,14 @@ public void Dispose()
             this.Dispose(true);
         }
 
+        /// <summary>
+        /// Overloaded Implementation of Dispose.
+        /// </summary>
+        /// <param name="isDisposing">Boolean that indicates whether the method call comes from a Dispose method or from a finalizer.</param>
```

Comment:
Couple stepping back questions:
* Given this is protected, why would we want XML doc?
* Perhaps we want to rename it to `disposing` per guidance from the [official doc](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose)?

Reply:
when i removed the bypass, VS started to complain about the protected methods as well. About the rename, sure!

Reply:
just renamed to disposing. About the xml to protected, after i removed the rule, protected methods started to complain...